### ISSUE TITLE
ldap-user-manager not starting — sed: can't read /etc/ldap/ldap.conf: No such file or directory

### ISSUE THREAD

**Issue by fooness** [wheelybird/ldap-user-manager#170]
**Check the debugging logs**
Unfortunately, the container does not start up, therefore I don’t see any logs besides:

```
docker logs -f 13db1c59e103
sed: can't read /etc/ldap/ldap.conf: No such file or directory
sed: can't read /etc/ldap/ldap.conf: No such file or directory
sed: can't read /etc/ldap/ldap.conf: No such file or directory
[…]
```

**Version**
`wheelybird/ldap-user-manager:latest`

**Describe the bug**
`ldap-user-manager` does not start via docker-compose setup.

```
$ cat docker-compose.yml
version: "3"

services:

  traefik:
    image: traefik:latest
    container_name: traefik
    restart: unless-stopped
    command:
      - "--api.dashboard=true"
      - "--api.insecure=true"
      - "--providers.docker=true"
      - "--providers.docker.exposedbydefault=false"
      - "--entrypoints.web.address=:80"
      - "--entrypoints.web.http.redirections.entrypoint.to=websecure"
      - "--entrypoints.web.http.redirections.entrypoint.scheme=https"
      - "--entrypoints.websecure.address=:443"
      - "--entrypoints.ldap-tcp.address=:1389/tcp"
      - "--entrypoints.ldap-udp.address=:1389/udp"
      - "--entrypoints.ldaps.address=:1636/tcp"
      - "--certificatesresolvers.myresolver.acme.tlschallenge=true"
      #- "--certificatesresolvers.myresolver.acme.caserver=https://acme-staging-v02.api.letsencrypt.org/directory"
      - "--certificatesresolvers.myresolver.acme.email=letsencrypt@example.com"
      - "--certificatesresolvers.myresolver.acme.storage=/letsencrypt/acme.json"
    labels:
      traefik.enable: "true"
      traefik.http.routers.api.entrypoints: "websecure"
      traefik.http.routers.api.tls.certresolver: "myresolver"
      traefik.http.routers.api.rule: "Host(`traefik.example.com`)"
      traefik.http.routers.api.service: "api@internal"
    ports:
      - "80:80"
      - "443:443"
    volumes:
      - ./data/letsencrypt:/letsencrypt
      - /var/run/docker.sock:/var/run/docker.sock:ro

  openldap:
    image: bitnami/openldap:latest
    […]

  ldap-user-manager:
    image: wheelybird/ldap-user-manager:latest
    container_name: ldap-user-manager
    restart: unless-stopped
    depends_on:
      - traefik
      - openldap
    environment:
      #LDAP_URI: "ldaps://localhost"
      LDAP_URI: "ldaps://localhost:1636"
      LDAP_BASE_DN: "dc=example,dc=com"
      LDAP_ADMIN_BIND_DN: "cn=admin,dc=example,dc=com"
      LDAP_ADMIN_BIND_PWD: "admipassword"
      LDAP_ADMINS_GROUP: "admins"
      #
      # […]
      #
      SERVER_PATH: "/"
      #SERVER_PORT: "80 & 443"
      SERVER_PORT: "8393 & 8636"
      NO_HTTPS: "false"
      #SERVER_KEY_FILENAME: "server.key"
      #SERVER_CERT_FILENAME: "server.crt"
      #CA_CERT_FILENAME: "ca.crt"
      #SESSION_TIMEOUT: "10 minutes"
      #
      # […]
      #
      LDAP_USER_OU: "users"
      LDAP_GROUP_OU: "readers"
      LDAP_REQUIRE_STARTTLS: "true"
      LDAP_IGNORE_CERT_ERRORS: "false"
      LDAP_TLS_CACERT: "/etc/ssl/certs/ca-certificates.crt"
      #
      # […]
      #
      LDAP_DEBUG: "true"
      LDAP_VERBOSE_CONNECTION_LOGS: "true"
      SESSION_DEBUG: "true"
      #SMTP_LOG_LEVEL: "0"
    labels:
      traefik.enable: "true"
      traefik.http.routers.ldap-user-manager.entrypoints: "websecure"
      traefik.http.routers.ldap-user-manager.tls.certresolver: "myresolver"
      traefik.http.routers.ldap-user-manager.rule: "Host(`ldap-user-manager.example.com`)"
    ports:
      - "8389:80"
      - "8636:443"

networks:
  default:
    driver: bridge
```

**Comment by wheelybird** [comment]
Hi.  Are you running this on an ARM platform?

**Comment by fooness** [comment]
> Hi. Are you running this on an ARM platform?

No; it’s a virtual machine running Ubuntu 22.04 LTS …

**Comment by fooness** [comment]
Thank you @wheelybird for your answer; I tried pulling the new image, unfortunately still the same behavior and errors.

**Comment by wheelybird** [comment]
I think somehow the change to fix that didn't get pushed to v1.8.  But it should be fixed in v1.9.  Hopefully.
